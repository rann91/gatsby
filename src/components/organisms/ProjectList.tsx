/* @jsx jsx */
import { graphql, useStaticQuery } from 'gatsby'
import { Box, jsx, Container } from 'theme-ui'
import {
  Strapi_ComponentContentProjectList,
  Strapi_Project
} from '../../typings/graphql'
import Section from '../atoms/Section'
import Headings from '../molecules/Headings'
import ProjectTeaser from '../molecules/ProjectTeaser'

const ProjectList = ({
  title,
  subtitle
}: Strapi_ComponentContentProjectList) => {
  const data = useStaticQuery<{
    strapi: {
      projects: Strapi_Project[]
    }
  }>(graphql`
    query {
      strapi {
        projects(sort: "createdAt:desc") {
          id
          title
          slug
          category
          image {
            ...ImageFluid
          }
        }
      }
    }
  `)
  return (
    <Section hasTitle={!!title}>
      <Container>
        <Headings {...{ title, subtitle }} />
        <Box
          sx={{
            '* + &': {
              paddingTop: [0, null, null, 5]
            }
          }}>
          {data.strapi.projects.map(project => (
            <Box key={project.id} p={5} mb={[0, null, null, 3]}>
              <ProjectTeaser {...project} />
            </Box>
          ))}
        </Box>
      </Container>
    </Section>
  )
}

export default ProjectList
