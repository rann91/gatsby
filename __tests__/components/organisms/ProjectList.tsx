import { render } from '@testing-library/react'
import { useStaticQuery } from 'gatsby'
import React from 'react'
import ProjectList from '../../../src/components/organisms/ProjectList'
import component from '../../fixtures/component-project-list.json'
import data from '../../fixtures/project-list.json'

beforeEach(() => {
  ;(useStaticQuery as jest.Mock).mockImplementationOnce(() => data)
})

describe('ProjectList', () => {
  it('renders', () => {
    const { container } = render(<ProjectList {...component} />)

    expect(container).toMatchSnapshot()
  })
})
