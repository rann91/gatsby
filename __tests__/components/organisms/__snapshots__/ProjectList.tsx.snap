// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectList renders 1`] = `
<div>
  <section
    class="css-1ptauik"
    data-testid="project-list"
  >
    <div
      class="css-1yrmf69"
    >
      <div
        class="css-1mpaekt-HeadingGroup"
      >
        <div
          class="css-146vepc-HeadingGroup"
          data-testid="subtitle"
        >
          Our work
        </div>
        <h2
          class="css-16nj6cn-HeadingGroup"
          data-testid="title"
        >
          Team projects
        </h2>
      </div>
      <div
        class="css-g58dgl-ProjectList"
      >
        <div
          class="css-1jva24b"
        >
          <a
            href="/work/project-1"
          >
            <div
              class="css-i9l3dq"
            >
              <div
                class="css-r01jhj"
              />
              <div
                class="css-1sr1905"
              >
                <div
                  class=" gatsby-image-wrapper"
                  style="position: relative; overflow: hidden;"
                >
                  <div
                    aria-hidden="true"
                    style="width: 100%; padding-bottom: 66.66666666666667%;"
                  />
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwAAAwEAAAAAAAAAAAAAAAAAAAMEBf/EABYBAQEBAAAAAAAAAAAAAAAAAAABAv/aAAwDAQACEAMQAAABRpZlrc4gj//EABkQAAMBAQEAAAAAAAAAAAAAAAECEQAEMf/aAAgBAQABBQJa2MTFnvP6SczG/wD/xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAEDAQE/AT//xAAUEQEAAAAAAAAAAAAAAAAAAAAQ/9oACAECAQE/AT//xAAYEAACAwAAAAAAAAAAAAAAAAAAAREgIf/aAAgBAQAGPwJKEZX/xAAbEAACAgMBAAAAAAAAAAAAAAAAAREhMUGREP/aAAgBAQABPyF37DOxETySk0LkXr75v//aAAwDAQACAAMAAAAQ8/8A/8QAFxEBAAMAAAAAAAAAAAAAAAAAAAERMf/aAAgBAwEBPxDVQ//EABcRAAMBAAAAAAAAAAAAAAAAAAABESH/2gAIAQIBAT8QWFZ//8QAGxABAQEAAwEBAAAAAAAAAAAAAREAITFRYaH/2gAIAQEAAT8QPcwqGxdEPfD2vi5qAt4PmBS7afmHxbJ6uIrGl3//2Q=="
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms;"
                  />
                  <picture>
                    <source
                      sizes="(max-width: 1920px) 100vw, 1920px"
                      srcset="/static/469dfe6e19619eaa5cde33d2c2e831a4/9104c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 480w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/a6352/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 960w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/d8255/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 1920w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/af56c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 2880w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/27fce/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 3840w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/41435/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 5371w"
                    />
                    <img
                      alt="Glasses on a surface"
                      loading="lazy"
                      sizes="(max-width: 1920px) 100vw, 1920px"
                      src="/static/469dfe6e19619eaa5cde33d2c2e831a4/d8255/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg"
                      srcset="/static/469dfe6e19619eaa5cde33d2c2e831a4/9104c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 480w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/a6352/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 960w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/d8255/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 1920w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/af56c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 2880w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/27fce/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 3840w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/41435/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 5371w"
                      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
                    />
                  </picture>
                  <noscript>
                    &lt;picture&gt;&lt;source srcset="/static/469dfe6e19619eaa5cde33d2c2e831a4/9104c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 480w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/a6352/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 960w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/d8255/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 1920w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/af56c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 2880w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/27fce/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 3840w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/41435/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 5371w" sizes="(max-width: 1920px) 100vw, 1920px" /&gt;&lt;img loading="lazy" sizes="(max-width: 1920px) 100vw, 1920px" srcset="/static/469dfe6e19619eaa5cde33d2c2e831a4/9104c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 480w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/a6352/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 960w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/d8255/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 1920w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/af56c/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 2880w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/27fce/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 3840w,
/static/469dfe6e19619eaa5cde33d2c2e831a4/41435/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg 5371w" src="/static/469dfe6e19619eaa5cde33d2c2e831a4/d8255/work_hand_book_flower_color_brown_1269173_pxhere_com_e819fdfbe5.jpg" alt="Glasses on a surface" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                  </noscript>
                </div>
              </div>
            </div>
          </a>
          <h3
            class="css-1j8rfyk-ProjectTeaser"
          >
            <a
              class="css-1tie34l-ProjectTeaser"
              href="/work/project-1"
            >
              Project #1
            </a>
          </h3>
          <p
            class="css-185gnmu-ProjectTeaser"
          >
            Art Direction
          </p>
        </div>
        <div
          class="css-1jva24b"
        >
          <a
            href="/work/project-2"
          >
            <div
              class="css-i9l3dq"
            >
              <div
                class="css-r01jhj"
              />
              <div
                class="css-1sr1905"
              >
                <div
                  class=" gatsby-image-wrapper"
                  style="position: relative; overflow: hidden;"
                >
                  <div
                    aria-hidden="true"
                    style="width: 100%; padding-bottom: 66.45833333333333%;"
                  />
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwAAAwEAAAAAAAAAAAAAAAAAAAIEBf/EABYBAQEBAAAAAAAAAAAAAAAAAAEAA//aAAwDAQACEAMQAAABrfFU0vMsb//EABkQAAMBAQEAAAAAAAAAAAAAAAECAwAREv/aAAgBAQABBQKPCvoah4yWZQ1GxoTv/8QAFREBAQAAAAAAAAAAAAAAAAAAABH/2gAIAQMBAT8BiP/EABURAQEAAAAAAAAAAAAAAAAAAAAR/9oACAECAQE/AYj/xAAYEAEBAAMAAAAAAAAAAAAAAAAAERAhgf/aAAgBAQAGPwKuXEVt/8QAGRABAQEBAQEAAAAAAAAAAAAAAQARITGh/9oACAEBAAE/IcMEnx3fhEGHpscHhKUMnvbf/9oADAMBAAIAAwAAABDX/wD/xAAVEQEBAAAAAAAAAAAAAAAAAAABEP/aAAgBAwEBPxAj/8QAFxEAAwEAAAAAAAAAAAAAAAAAAAERIf/aAAgBAgEBPxC6LB//xAAZEAEBAQEBAQAAAAAAAAAAAAABESEAMZH/2gAIAQEAAT8QswCePXDgsmQz70wUHXMDpQvTYMkdy3jSQJbM7//Z"
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms;"
                  />
                  <picture>
                    <source
                      sizes="(max-width: 1920px) 100vw, 1920px"
                      srcset="/static/d2a3cba7e42ebf5a5d9296f8be6fe929/9104c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 480w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/a6352/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 960w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/d8255/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 1920w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/af56c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 2880w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/27fce/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 3840w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/1b38f/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 5014w"
                    />
                    <img
                      alt="Coca cola balanced on a hand"
                      loading="lazy"
                      sizes="(max-width: 1920px) 100vw, 1920px"
                      src="/static/d2a3cba7e42ebf5a5d9296f8be6fe929/d8255/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg"
                      srcset="/static/d2a3cba7e42ebf5a5d9296f8be6fe929/9104c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 480w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/a6352/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 960w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/d8255/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 1920w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/af56c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 2880w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/27fce/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 3840w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/1b38f/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 5014w"
                      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
                    />
                  </picture>
                  <noscript>
                    &lt;picture&gt;&lt;source srcset="/static/d2a3cba7e42ebf5a5d9296f8be6fe929/9104c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 480w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/a6352/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 960w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/d8255/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 1920w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/af56c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 2880w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/27fce/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 3840w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/1b38f/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 5014w" sizes="(max-width: 1920px) 100vw, 1920px" /&gt;&lt;img loading="lazy" sizes="(max-width: 1920px) 100vw, 1920px" srcset="/static/d2a3cba7e42ebf5a5d9296f8be6fe929/9104c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 480w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/a6352/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 960w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/d8255/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 1920w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/af56c/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 2880w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/27fce/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 3840w,
/static/d2a3cba7e42ebf5a5d9296f8be6fe929/1b38f/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg 5014w" src="/static/d2a3cba7e42ebf5a5d9296f8be6fe929/d8255/coca_cola_drink_vietnam_project_hand_art_1455157_pxhere_com_72eb58bdd6.jpg" alt="Coca cola balanced on a hand" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                  </noscript>
                </div>
              </div>
            </div>
          </a>
          <h3
            class="css-1j8rfyk-ProjectTeaser"
          >
            <a
              class="css-1tie34l-ProjectTeaser"
              href="/work/project-2"
            >
              Project #2
            </a>
          </h3>
          <p
            class="css-185gnmu-ProjectTeaser"
          >
            Product Design
          </p>
        </div>
        <div
          class="css-1jva24b"
        >
          <a
            href="/work/project-3"
          >
            <div
              class="css-i9l3dq"
            >
              <div
                class="css-r01jhj"
              />
              <div
                class="css-1sr1905"
              >
                <div
                  class=" gatsby-image-wrapper"
                  style="position: relative; overflow: hidden;"
                >
                  <div
                    aria-hidden="true"
                    style="width: 100%; padding-bottom: 70%;"
                  />
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAAOABQDASIAAhEBAxEB/8QAFwAAAwEAAAAAAAAAAAAAAAAAAAIEBf/EABUBAQEAAAAAAAAAAAAAAAAAAAAB/9oADAMBAAIQAxAAAAGOfTZXJSP/xAAbEAACAwADAAAAAAAAAAAAAAABAwACBBEUQf/aAAgBAQABBQLUuzNHXsIHW48sxcJn/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAwEBPwE//8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAgEBPwE//8QAHhAAAQQBBQAAAAAAAAAAAAAAAQACEHERAxITIkH/2gAIAQEABj8CJ0xupdi1tleRjjzcf//EABwQAQACAgMBAAAAAAAAAAAAAAEAESExEEFRYf/aAAgBAQABPyFAIQzw0AgGhUHOlry4MBZXfUVrWPk//9oADAMBAAIAAwAAABCg/wD/xAAVEQEBAAAAAAAAAAAAAAAAAAAAEf/aAAgBAwEBPxCI/8QAFREBAQAAAAAAAAAAAAAAAAAAABH/2gAIAQIBAT8Qqv/EAB0QAQEAAgMAAwAAAAAAAAAAAAERACExQVFhgZH/2gAIAQEAAT8Qd74hQZEvHWH3lAX8LlHoFR3DnFkBciDJO6B7HkzYDRDQ18+5/9k="
                    style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms;"
                  />
                  <picture>
                    <source
                      sizes="(max-width: 1920px) 100vw, 1920px"
                      srcset="/static/fba2fa424fd724cdc60e29346a139901/9104c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 480w,
/static/fba2fa424fd724cdc60e29346a139901/a6352/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 960w,
/static/fba2fa424fd724cdc60e29346a139901/d8255/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 1920w,
/static/fba2fa424fd724cdc60e29346a139901/af56c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 2880w,
/static/fba2fa424fd724cdc60e29346a139901/27fce/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 3840w,
/static/fba2fa424fd724cdc60e29346a139901/10891/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 4524w"
                    />
                    <img
                      alt="iMac on a desk"
                      loading="lazy"
                      sizes="(max-width: 1920px) 100vw, 1920px"
                      src="/static/fba2fa424fd724cdc60e29346a139901/d8255/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg"
                      srcset="/static/fba2fa424fd724cdc60e29346a139901/9104c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 480w,
/static/fba2fa424fd724cdc60e29346a139901/a6352/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 960w,
/static/fba2fa424fd724cdc60e29346a139901/d8255/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 1920w,
/static/fba2fa424fd724cdc60e29346a139901/af56c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 2880w,
/static/fba2fa424fd724cdc60e29346a139901/27fce/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 3840w,
/static/fba2fa424fd724cdc60e29346a139901/10891/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 4524w"
                      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;"
                    />
                  </picture>
                  <noscript>
                    &lt;picture&gt;&lt;source srcset="/static/fba2fa424fd724cdc60e29346a139901/9104c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 480w,
/static/fba2fa424fd724cdc60e29346a139901/a6352/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 960w,
/static/fba2fa424fd724cdc60e29346a139901/d8255/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 1920w,
/static/fba2fa424fd724cdc60e29346a139901/af56c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 2880w,
/static/fba2fa424fd724cdc60e29346a139901/27fce/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 3840w,
/static/fba2fa424fd724cdc60e29346a139901/10891/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 4524w" sizes="(max-width: 1920px) 100vw, 1920px" /&gt;&lt;img loading="lazy" sizes="(max-width: 1920px) 100vw, 1920px" srcset="/static/fba2fa424fd724cdc60e29346a139901/9104c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 480w,
/static/fba2fa424fd724cdc60e29346a139901/a6352/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 960w,
/static/fba2fa424fd724cdc60e29346a139901/d8255/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 1920w,
/static/fba2fa424fd724cdc60e29346a139901/af56c/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 2880w,
/static/fba2fa424fd724cdc60e29346a139901/27fce/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 3840w,
/static/fba2fa424fd724cdc60e29346a139901/10891/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg 4524w" src="/static/fba2fa424fd724cdc60e29346a139901/d8255/computer_work_table_keyboard_technology_mouse_931790_pxhere_com_3414eb5be7.jpg" alt="iMac on a desk" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
                  </noscript>
                </div>
              </div>
            </div>
          </a>
          <h3
            class="css-1j8rfyk-ProjectTeaser"
          >
            <a
              class="css-1tie34l-ProjectTeaser"
              href="/work/project-3"
            >
              Project #3
            </a>
          </h3>
          <p
            class="css-185gnmu-ProjectTeaser"
          >
            Animation
          </p>
        </div>
      </div>
    </div>
  </section>
</div>
`;
